how-to/integrate-with-bloomberg-basic/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "integrate-with-bloomberg-basic",
+	"version": "1.0.0",
+	"description": "OpenFin Workspace - Example shows how to connect to a Bloomberg terminal and issue basic commands to drive it",
+	"main": "index.js",
+	"scripts": {
+		"dos": "npm --prefix ../common/ run dos",
+		"build": "npm run build-server & npm run build-client",
+		"build-server": "tsc --project ./server",
+		"build-client": "webpack build --config ./client/webpack.config.js --mode=development",
+		"build-client-prod": "webpack build --config ./client/webpack.config.js --mode=production",
+		"start": "npm run server",
+		"client": "npm --prefix ../common/ run client",
+		"server": "node ./server/build/index.js",
+		"kill": "npm --prefix ../common/ run kill",
+		"setup": "cd ../../ && npm install && cd how-to/integrate-with-bloomberg-basic && npm run build && npm --prefix ../common/ run build",
+		"validate": "npm --prefix ../../ run validate"
+	},
+	"keywords": [
+		"bloomberg",
+		"integration"
+	],
+	"author": "ameet.jayawant@openfin.co",
+	"license": "SEE LICENSE IN LICENSE.MD",
+	"dependencies": {
+		"@openfin/bloomberg": "^1.1.0",
+		"@openfin/core": "^29.73.14",
+		"@openfin/workspace": "11.0.6",
```

Comment:
Core, Workspace and Workspace platform need to be updated to the relevant versions for version 12 of workspace.